frontend/src/styles/variables.css

```
@@ -209,75 +330,48 @@
   --z-tooltip: 1100;
   --z-toast: 1200;
   
-  /* ========== 布局尺寸 Layout ========== */
+  /* ===== Z-index 层级 ===== */
+  --z-dropdown: 100;
```

Stylelint (17.3.0)

[error] Unexpected duplicate "--z-dropdown" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)